openstack-user-data.txt

```diff
@@ -0,0 +1,16 @@
+#cloud-config
+bootcmd:
+  - touch /tmp/init.out
+system_info:
+  default_user:
+    name: ubuntu
+packages:
+ - python-virtualenv
+ - git
+runcmd:
+ - su - -c '(set -x ; git clone -b wip-6502-openstack-v2 http://github.com/dachary/teuthology && cd teuthology && ./bootstrap install)' ubuntu >> /tmp/init.out 2>&1
```

Comment:
So, what if we want to use a different teuthology branch? Will that be possible somehow?